Fabric.Identity.API/Quickstart/Account/AccountController.cs

```diff
@@ -237,9 +242,27 @@ public async Task<IActionResult> ExternalLoginCallback(string returnUrl)
             {
                 userIdClaim = claims.FirstOrDefault(x => x.Type == ClaimTypes.NameIdentifier);
             }
+
             if (userIdClaim == null)
             {
-                throw new Exception("Unknown userid");
+                throw new MissingUserClaimException(ExceptionMessageResources.MissingUserClaimMessage);
+            }
+
+            var schemaItem = info.Properties.Items.FirstOrDefault(i => i.Key == "scheme");
+
+            if (_appConfiguration.AzureAuthenticationEnabled && schemaItem.Value == FabricIdentityConstants.AuthenticationSchemes.Azure)
```

Comment:
Do we need to check schemaItem for null here?

Reply:
nope.  schemaItem is actually a key/value pair.  A dictionary returns a "","" pair if first or default is not found.